src/cli/tc.rs

```diff
@@ -163,15 +166,8 @@ pub async fn init(folder_path: &str) -> Result<()> {
     Ok(())
 }
 
-pub fn display_schema(blueprint: &Blueprint) {
-    Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
-    let sdl = blueprint.to_schema();
-    Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
-}
-
-fn display_config(config: &Config, n_plus_one_queries: bool) {
-    let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
-    Fmt::display(Fmt::table(seq));
+fn display(s: String) {
+    println!("{}", s);
```

Comment:
Why aren't we using `Fmt::display`? All the color coding is gone.

Reply:
this is the code from Fmt::display, I think color coding still works for me